lib/conjur/conjur_config.rb

```diff
@@ -0,0 +1,28 @@
+module Conjur
+  # Reads application config from a YAML file on disk, as well as env vars
+  # prefixed with CONJUR_ then serves as a single point to access configuration
+  # from the Conjur application code.
+  class ConjurConfig < Anyway::Config
+    config_name :config
+
+    # Env vars prefixed w/ CONJUR_ overwrite values loaded from file. Note that
+    # Anyway Config caches these values at the class level and they will not be
+    # picked up by a call to `reload` unless you first run `Anyway.env.clear`.
+    env_prefix :conjur
+
+    attr_config(
+      # Read TRUSTED_PROXIES before default to maintain backwards compatibility
+      trusted_proxies: (ENV['TRUSTED_PROXIES'] || "")
+    )
+
+    def attribute_sources
+      sources = {}
+
+      to_source_trace.each do |k, v|
+        sources[k.to_sym] = v[:source][:type]
+      end
+
+      sources
```

Comment:
Something like this should work (ie, this line would be the entire method body):

```ruby
to_source_trace.map { |k,v| [ k.to_sym, v[:source][:type] ] }.to_h